test/stage2/compare_output.zig

```diff
@@ -23,11 +23,6 @@ pub fn addCases(ctx: *TestContext) !void {
 
         case.addError("", &[_][]const u8{":1:1: error: no entry point found"});
 
-        case.addError(
```

Comment:
I was thinking for stage2 we could organize by feature (and by testing the incremental compilation, which this is doing by making sure it can go from an error to a successfully building program). I'll do that when merging to show you what I mean.